pkg/agent/route/route_linux.go

```diff
@@ -340,6 +354,17 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet string) *bytes
 
 	writeLine(iptablesData, "*nat")
 	writeLine(iptablesData, iptables.MakeChainLine(antreaPostRoutingChain))
+	// Egress rule must be inserted before the default masquerade rule.
+	for snatMark, snatIP := range snatMarkToIP {
+		// Cannot reuse snatRuleSpec to generate the rule as it doesn't have "`" in the comment.
+		writeLine(iptablesData, []string{
+			"-A", antreaPostRoutingChain,
+			"-m", "comment", "--comment", `"Antrea: snat pod to external packets"`,
```

Comment:
Do we follow the convention of "SNAT" and "Pod" in iptables comments?

Reply:
done, and updated other comments too.